routes/web-public.php

```diff
@@ -63,5 +63,6 @@
         Route::get('/data-protection-policy')->uses('PolicyPagesController@viewDPP')->name('data-protection');
         Route::get('/branding-guidelines')->uses('PolicyPagesController@viewBranding')->name('branding');
         Route::get('/streaming-guidelines')->uses('PolicyPagesController@viewStreaming')->name('streaming');
+        Route::get('/training-process/s1-syllabus')->uses('PolicyPagesController@viewS1Syllabus')->name('s1-syllabus');
```

Comment:
This is probably what you were aiming for
```suggestion
        Route::group([
            'as' => 'training-process.',
            'prefix' => 'training-process',
        ], function () {
            Route::get('/s1-syllabus')->uses('PolicyPagesController@viewS1Syllabus')->name('s1-syllabus');
        });
```